app/javascript/common/helpers.js

```diff
@@ -1,3 +1,6 @@
 // todo: expose this from patternfly-react instead?
 export const getDisplayName = Component =>
   Component.displayName || Component.name || 'Component';
+
+export const pluralize = (count, string) =>
```

Comment:
Please don't ever use this `pluralize` :)

Won't work with i18n at all.

Reply:
@priley86 Grammar rules vary from language to language, so what defines 'pluralization' in English (adding an `s` or `es` in most cases) does not apply to all languages.

To keep it simple, lets just go with - 
`__('Source Datastore')` for one and 
`__('Source Datastores')` for many

Reply:
this is a SUPER neat idea though! Like this idea... if complete sentences make the world an easier place... we really should just use the react-intl helper thing... (especially since we already have the dep)


[we can use the react-intl formatted plural thing](https://github.com/yahoo/react-intl/wiki/Components#formattedplural)

```
<FormattedPlural value={sourceDatastoreCount} one=__('Source Datastore') other=__('Source Datastores') />
```